packages/backend-test-utils/api-report.md

```diff
@@ -272,6 +283,29 @@ export namespace mockServices {
       ) => ServiceMock<RootConfigService>;
   }
   // (undocumented)
+  export function rootEventAuditor(
+    options?: rootEventAuditor.Options,
+  ): EventAuditorService;
+  // (undocumented)
+  export namespace rootEventAuditor {
+    // (undocumented)
+    export type Options = {
+      level?: 'none' | 'error' | 'warn' | 'info' | 'debug';
+    };
+    const // (undocumented)
+      factory: (
+        options?: Options | undefined,
+      ) => ServiceFactory<
+        EventAuditorService,
+        'plugin',
+        'singleton' | 'multiton'
```

Comment:
we want these three to end up as `RootEventAuditorService`, `'root'`, and `'singleton'`.